src/test/dynamo/util.ts

```diff
@@ -51,19 +51,23 @@ export async function createJayZ(): Promise<JayZ> {
 // DynamoDB has a 400kb Item limit w/ a 1MB response size limit
 // Thus 25 song items comprise at least 100kb * 25 = ~2.5MB of data
 // i.e. at least 3 pages. Note that data encrypted with JayZ is significantly larger
-export async function createSongs(
-  db: Beyonce,
-  n: number = 25
-): Promise<Song[]> {
-  const mp3 = crypto.randomBytes(100_000)
-  const songs: Song[] = [...Array(n).keys()].map((songId) =>
-    SongModel.create({
-      musicianId: "1",
-      id: songId.toString(),
-      title: `Song ${songId}`,
-      mp3
-    })
-  )
-  await Promise.all(songs.map((song) => db.put(song)))
+const mp3 = crypto.randomBytes(100_000)
```

Comment:
The changes to this method here are an optimization that cuts the test execution time down.